Cargo.toml

```diff
@@ -0,0 +1,53 @@
+[workspace]
+members = [
+  "frb_codegen",
+  "frb_rust",
+  "frb_macros",
+  "frb_example/pure_dart/rust",
+  "frb_example/pure_dart_multi/rust",
+  "frb_example/with_flutter/rust",
+]
+# TODO: refactor with glob see https://github.com/rust-lang/cargo/issues/6009
+# used to ignore corrosion checkout files in CI tests (Linux / Windows)
+exclude = [
+  "corrosion/test/cpp2rust/rust",
```

Comment:
Would removing corrosion after installation help make this smaller? I think it's confusing for our workspace to reference unrelated crates when we're not using glob members.

Reply:
good idea, let's try

Reply:
Given cargo is run right away when installing Corrosion, I can't think of a way to do it atm @Desdaemon 

<img src="https://user-images.githubusercontent.com/21016014/192306081-f5620042-297c-4362-951c-710dc553dd1e.png" width="800" />

Reply:
resolved across e789d03, 0e2e2ab and e34ed00